features/step_definitions/steps.rb

```diff
@@ -17,14 +18,22 @@
 end
 
 Given(/^a page with the HTML:$/) do |string|
-  @page = Capybara.string(string.to_s)
+  WebServer.ensure_running_on(54321)
+  WebServer.write_page "scenario.html", "<html><body>#{string}</body></html>"
+  browser.visit 'http://localhost:54321/scenario.html'
```

Comment:
Actually I don't really understand your question. I assumed you meant: put 54321 in a configuration file somewhere. I don't want to do that until I have a good reason to.